packages/web-twig/src/Resources/components/FileUploader/__tests__/__fixtures__/fileUploaderInput.twig

```diff
@@ -24,6 +24,20 @@
     multiple
 />
 
+<!-- Render with dynamic multiple attribute -->
```

Comment:
I'm wondering why "dynamic" 🤔 

```suggestion
<!-- Render without `multiple` attribute -->
```

or

```suggestion
<!-- Do not render `multiple` attribute when false -->
```

?

Reply:
This was reverted.